benchmarks/fibonacci/docker-compose.yaml

```diff
@@ -0,0 +1,81 @@
+version: "3.9"
+services:
+  ## Python
+  fibonacci-python:
+    image: davidschall/fibonacci-python:v1
+    container_name: fibonacci-python
+    build: 
+      context: ./
+      dockerfile: ./python/Dockerfile.python
+    ports:
+    - published: 50061
+      target: 50051
+
+  # fibonacci-cython:
```

Comment:
Please remove all other python interpreters but the normal python one. We dont need to add Cython and PyPy for now.